js/ui/notificationDaemon.js

```diff
@@ -880,4 +904,13 @@ const NotificationDaemon = new Lang.Class({
         this._fdoNotificationDaemon = new FdoNotificationDaemon();
         this._gtkNotificationDaemon = new GtkNotificationDaemon();
     },
+
+    get fdo() {
+        return this._fdoNotificationDaemon;
+    },
+
+    get gtk() {
+        return this._gtkNotificationDaemon;
+    },
```

Comment:
Now that we are using `messageTray` to check notifications, is it necessary to expose this?